robocopkaka/hospital
# #160663419 Setup tests

#### What does this PR do?
Setup tests
#### Description of Task to be completed?
* Add gems required for testing
* Add config for testing
* Add tests for controllers
* Add tests for models
* Integrate test coverage took (SimpleCov)
#### How should this be manually tested?
* Clone and setup this repo locally
* Run `rspec spec` to run all the tests
* Right-click on `coverage/index.html` and copy the full file path and paste in your browser
* You should see the test coverage for the project
#### What are the relevant pivotal tracker stories?
[#160663419](https://www.pivotaltracker.com/story/show/160663419)
#### Any background context you want to add?
N/A
